src/solc_0.7/common/BaseWithStorage/WithMetaTransaction.sol

```diff
@@ -17,7 +17,7 @@ contract WithMetaTransaction is WithAdmin {
     /// @param metaTransactionProcessor address that will be given/removed metaTransactionProcessor rights.
     /// @param processorType set the metaTransactionProcessor type
     function setMetaTransactionProcessor(address metaTransactionProcessor, uint8 processorType) public {
-        require(msg.sender == _admin, "only admin can setup metaTransactionProcessors");
+        require(msg.sender == _admin, "ADMIN_ACCESS_DENIED");
```

Comment:
Shoudl we not use the same string for both onlyAdmin and require(msg.sender == _admin, "ADMIN_ACCESS_DENIED");
or maybe we should use the modifier here ?

Reply:
indeed, the modifier would be the best approach now.

Reply:
I haven't got into the habit of using modifiers consistently yet. I expect we'll find more cases where we could apply them as we go.